src/index.html

```diff
@@ -8,10 +8,159 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?:
+          <input
+            type="number"
+            name="Age(year)"
+            placeholder="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="Birth Date"
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) p.4
```suggestion
            name="birthDate"
```